### DISCUSSION TITLE
Erro ao ródar o VSCODE

### DISCUSSION THREAD

**Discussion by Anderson-Mendes-de-Almeida** [balta-io/forum#663] [Tira Dúvidas] [answered]
Cara to seguindo a aula e ao abrir o VSCODE coloque 4 linhas de código que o Balta ensina, mas ele ja vem com erro na linha 4                                        =>var app:WebApplication = builder.Build();
![Erro-VSCODE-1](https://github.com/user-attachments/assets/269b116f-ac4e-4530-9c79-4cd3a613edea)
![Erro-VSCODE-2](https://github.com/user-attachments/assets/76f58e1f-753c-4159-893a-55f591c36ac0)

**Comment by andrebaltieri** [comment, marked as answer]
@Anderson-Mendes-de-Almeida esse `app` que aparece é o tipo da variável, um recurso que minha IDE oferece e não precisa ser digitado.

Recomendo que comece os estudos seguindo nosso guia:
https://balta.io/carreiras

Segue os cursos nessa ordem que chegará no Fullstack bem preparado 🚀
